services/auth.js

```diff
@@ -42,11 +41,13 @@ const authService = {
       firstName,
       lastName,
       email,
-      password: hashedPassword,
-      activationLink
+      password: hashedPassword
     })
 
-    await sendEmail(email, emailSubject.EMAIL_CONFIRMATION, { activationLink, email, firstName })
+    const confirmToken = tokenService.generateConfirmToken({ id: user._id, isEmailConfirmed: false })
```

Comment:
actually you don't need to add `isEmailConfirmed` field to [payload

Reply:
fixed